frontend/yarn.lock

```diff
@@ -8023,6 +8087,21 @@ kind-of@^6.0.0, kind-of@^6.0.2:
   version "6.0.2"
   resolved "https://registry.yarnpkg.com/kind-of/-/kind-of-6.0.2.tgz#01146b36a6218e64e58f3a8d66de5d7fc6f6d051"
 
+kubevirt-web-ui-components@^0.1.34-4:
+  version "0.1.34-7"
+  resolved "https://registry.yarnpkg.com/kubevirt-web-ui-components/-/kubevirt-web-ui-components-0.1.34-7.tgz#d63ef14fad42cc1b197275f52f48429c568dac6d"
+  integrity sha512-q7UQqRdRz80x5SG1entoOWurHKIAmt7XWh6OaCoTSE8YJwjhLny4TgjwW441eFOp+OUTqc0p5vNpBxiQc27QKw==
+  dependencies:
+    "@patternfly/react-console" "1.x"
+    classnames "2.x"
+    js-yaml "3.x"
+    lodash "4.x"
+    react-dnd "2.6.x"
+    react-dnd-html5-backend "5.0.x"
+    react-measure "2.x"
```

Comment:
We should look at removing this dependency from kubevirt-web-ui-components if we can. See discussion in #1591